### DISCUSSION TITLE
Are there nightly builds?

### DISCUSSION THREAD

**Discussion by StevenTCramer** [microsoft/fluentui-blazor#2067] [Q&A]
Things are moving fast with FluentUI, no pun intended.  Are there nightly builds of the nuget packages being deploy somewhere?  I'm a bit anxious to get the changes.  :)

**Comment by vnbaaij** [comment]
We are trying to get that set up again.

We had them before, but needed to change our build pipelines because of a security effort. Good news is we are now part of the actual .NET Azure DevOps organization. Bad news is we need to do some things, like scheduled runs, differently .But as said, working on it.

**Reply by StevenTCramer** [reply]
I was just looking at the pipelines couldn't see where you even publish the nuget packages.   I would be happy to help if I can.

**Reply by dvoituron** [reply]
We're working on publishing it in the NuGet.org public feed, with the `preview` tag.
Thanks for your help, but this package is generated from MS's internal server.

**Reply by vnbaaij** [reply]
The publishing to NuGet is a manual step. As we only do it like once every 2-3 weeks, I've not spent the time on automating it yet. It is on the list (somewhere)
